src/pages/Video.tsx

```diff
@@ -67,15 +67,17 @@ export default function Video() {
             }}
             mb={2}
           >
+            {/* @ts-ignore */}
```

Comment:
This thing starts throwing weird typing issues, not even sure which lib is acting weird...